### DISCUSSION TITLE
Documentation: How to interpret Auto Regression Parameter Plot

### DISCUSSION THREAD

**Discussion by winnieshen96** [ourownstory/neural_prophet#1161] [Q&A - get help using NeuralProphet] [answered]
**Is your feature request related to a problem? Please describe.**

Since compared to the original prophet library, there is now auto regression in neural prophet, so when plotting components of the model, there is auto regression plot. I find it difficult to understand what the auto regression plot means and how to interpret it as a business user. What is the y axis and how does the model compute the values?

**Describe the solution you'd like**

A documentation that gives examples with interpretation of the auto regression plot.

**Describe alternatives you've considered**

I tried to look through the entire plot_forecaster.py and forecaster.py to look for answers but soon it touches the different Nets calculation where it's harder to follow.

**Comment by ourownstory** [comment]
Thank You @winnieshen96 for bringing this to our attention. I think this is a great suggestion.

We could address this by adding documentation:
- docstring in `plot_parameters`: explain meaning of each type of model parameter plot.
- tutorial giving an overview over the `plot_parameters` plots and their meaning.

**Comment by ourownstory** [comment, marked as answer]
@winnieshen96 
This may not answer your question directly, but maybe this notebook can help:
https://github.com/ourownstory/neural_prophet/blob/main/tutorials/feature-use/autoregression_yosemite_temps.ipynb

some further resources:
* https://en.wikipedia.org/wiki/Autoregressive_model
* https://arxiv.org/abs/1911.12436 (section 3.2)
* https://arxiv.org/abs/2111.15397 (section 2.1.3)
